linux_pipeline/Jenkinsfile_lis_rpm_hyperv_lisav2

```diff
@@ -0,0 +1,263 @@
+            name: 'EXECUTION_TAG',
+            defaultValue: "",
+            description: '']
+        ]
+    ]
+])
+
+// defines deploy scenarios coverage - platform independent
+def nodesMap = ["sriov":"rhel_7.3,rhel_7.4,centos_7.4,centos_7.3,rhel_7.3_gen2vm,rhel_7.5,centos_7.5,", \
+                "ws2012":"centos_6.8_32bit,centos_6.8_x64,centos_6.9_32bit,rhel_6.7_gen2vm,rhel_6.8_x64,", \
+                "ws2012r2":"rhel_6.4_32bit,rhel_6.4_x64,rhel_6.5,rhel_6.6_x64,rhel_6.6_32bit, \
+                    centos_6.4_x64,centos_6.5_x64,rhel_6.9_x64,", \
+                "ws2016":"oracle_6.5_rhck,oracle_6.9_rhck,oracle_7.4_rhck,oracle_7.0_rhck,centos_7.0_x64, \
+                    centos_7.0_gen2vm,centos_7.2_x64,rhel_7.0,rhel_7.1,rhel_7.2,rhel_7.6,rhel_6.10_x64,"]
+
+// defines host version mapping for functional test validation
+def validationNodesMap = ["ws2016":"rhel_6.10_x64,rhel_7.4,centos_6.5_x64,centos_7.3,centos_7.2_x64,rhel_7.5,centos_7.5,", \
+                          "ws2012r2":"rhel_6.6_x64,centos_7.4_gen2vm,rhel_6.4_32bit,rhel_7.0,centos_6.4_x64,rhel_6.9_x64,", \
+                          "ws2012_fvt":"centos_6.8_x64,", \
+                          "ws2012_bvt":"rhel_7.1,"]
+
+def supportedDistros = nodesMap["ws2012"] + nodesMap["ws2012r2"] + nodesMap["ws2016"] + nodesMap["sriov"]
+
+def RunPowershellCommand(psCmd) {
+    bat "powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command \"[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;$psCmd;EXIT \$global:LastExitCode\""
+}
+
+def getDistros (distros, supportedDistros) {
+    def validatedDistros = []
+    suppList = supportedDistros.split(",")
+    distros = distros.split(",")
+
+    for (distro in distros) {
+        distroType = distro.split("_")[0]
+        distroVersion = distro.split("_")[1]
+
+        if (distroVersion.toLowerCase() == "all") {
+            for (suppDistro in suppList){
+                if (distroType.toLowerCase() == suppDistro.split("_")[0]) {
+                    validatedDistros << suppDistro
+                }
+            }
+        } else {
+            if (supportedDistros.contains(distro.toLowerCase())) {
+                validatedDistros << distro.toLowerCase()
+            }
+        }
+    }
+    return validatedDistros
+}
+
+def ErrorCount = 0
+def FVTDistros = "centos_6.8_x64,centos_7.2_x64,rhel_6.6_x64,centos_7.4_gen2vm,rhel_6.10_x64,rhel_7.4,rhel_7.5,"
+def BVTDistros = "centos_6.4_x64,rhel_7.1,rhel_6.4_32bit,rhel_6.9_x64,rhel_7.0,centos_6.5_x64,centos_7.3,centos_7.5,"
+def SRIOVDistros = "rhel_7.3,rhel_7.4,centos_7.4,centos_7.3,rhel_7.3_gen2vm,rhel_7.5,centos_7.5,"
+
+def CategoryAreaMap = ["Functional" : "LIS_DEPLOY,LIS,DYNAMIC_MEMORY,KVP,MIGRATION,NETWORK,SRIOV,STORAGE,BACKUP,CORE,FCOPY,KDUMP,PROD_CHECKPOINT,RUNTIME_MEMORY",\
+                "BVT":"BVT,CORE,NETWORK",\
+                "Stress":"stress" ]
+
+def TestAndDistroMap = ["Functional-LIS_DEPLOY" : "${supportedDistros}",\
```

Comment:
KVP, FCOPY and CORE should be part of BVT as well.
The current BVT tests are not doing much, and at least the above 3 areas are P0 on Hyper-V.